src/python/zquantum/qcbm/ansatz.py

```diff
@@ -26,7 +40,7 @@ def get_single_qubit_layer(n_qubits, params, single_qubit_gates):
     return output
 
 
-def get_all_topology(n_qubits, params, static_entangler):
+def get_all_topology(params, n_qubits, static_entangler):
```

Comment:
Is there a particular reason why n_qubits and params arguments have been switched everywhere? Fine with that for me, just curious to know :)

Reply:
I needed it to make it easily work with `build_ansatz_circuit` (or other function, I'm not 100% sure), which needed `params` to be passed as the first parameter.
This kind of issue will be solved with the ansatz interface :)